Automattic/wp-calypso
# Setting Up A/B Test for G Suite Upsell

This PR will set up an A/B test for G Suite upsell to get more users use G Suite upgrade.
See p7EOS0-3aB-p2 for more detail.

## Test plan

* Set the _gsuiteUpsell_ A/B test to _show_.
* Create a new site with a custom domain, not under wordpress.com.
* You should be able to see the G Suite upsell page after checking out the cart as follows:
![screen shot 2017-10-25 at 23 10 19](https://user-images.githubusercontent.com/212034/32003367-c1d5b7de-b9d9-11e7-8f3e-f6564ecf3203.png)
* The thank you page shows up when you click on the cancel link.
* If you want to upgrade your site with G Suite, click on the blue button at the footer, fill out the form and push the "Continue" button, then you will see the check out page again. If everything goes well, you will be at the thank you page with G Suite account setup message.
![screen shot 2017-10-25 at 23 36 52](https://user-images.githubusercontent.com/212034/32004904-e56133be-b9dd-11e7-9eeb-a04d6f8dae73.png)

cc @fditrapani @anneforbush
